app/src/main/java/com/example/mycleaninglog/dto/cleaningTask.kt

```diff
@@ -2,9 +2,9 @@ package com.example.mycleaninglog.dto
 
 import com.google.firebase.firestore.Exclude
 
-data class cleaningTask(var cleaningTaskName:String = "", var cleaningTaskId:String = "", var uniqueID:String = ""){
+data class Task(var TaskName:String = "", var TaskId:String = "", var uniqueID:String = ""){
```

Comment:
Task probably makes more sense than CleaningTask, as Task is more versatile.  Also, should be PascalCase.